### PR TITLE
Feature/simplify endpoints

### PR DESCRIPTION
**Pull request by albrazeau**
This PR makes it so that an entire RAS model does not need to be indexed when hitting the modelversion, isamodel, isgeospatial, and modeltype endpoints. This increases the speed of these endpoints tremendously.

### CODE REVIEW

**Review by ar-siddiqui — COMMENTED** [review]
Apart from one comment, my only other thought is that geospatialdata endpoint also builds a whole model and then apply the [following functions](https://github.com/Dewberry/mcat-ras/blob/e2a7c16654b6bf67802e3286385d0531d1dce48d/tools/model.go#L330) `getFlowData`, `getGeomData`, `getPlanData`, and then goes on retrieving geospatial data. I am not 100% sure but I don't think these functions are needed to retrieve geospatial data. They are needed for the index but not for geospatial data. These functions are time taking, basically, they do most of the work for the index endpoint and might not be needed for geospatial data.

I can investigate this more if required.

File: handlers/isamodel.go [hunk 1]
```diff
@@ -26,12 +28,23 @@ func IsAModel(fs *filestore.FileStore) echo.HandlerFunc {
...
-		rm, err := ras.NewRasModel(definitionFile, *fs)
-		if err != nil {
-			return c.JSON(http.StatusOK, false)
-		}
-		isIt := rm.IsAModel()
+		return c.JSON(http.StatusOK, isAModel(fs, definitionFile))
+	}
+}
+
+func isAModel(fs *filestore.FileStore, definitionFile string) bool {
```
[the remark this answers is not in the document]
Comment (albrazeau) [reply handlers/isamodel.go:35]: Resolved in [](https://github.com/Dewberry/mcat-ras/pull/13/commits/0fed072346c727220b5fc30a250da301e31ded76)[0fed072](https://github.com/Dewberry/mcat-ras/pull/13/commits/0fed072346c727220b5fc30a250da301e31ded76)

**Comment by albrazeau** [comment]
@ar-siddiqui I have simplified the Geospatial Data endpoint as well in [175662c](https://github.com/Dewberry/mcat-ras/pull/13/commits/175662cf18c63ce9484194c6e4f35f97ab2a62c2)

**Review by ar-siddiqui — APPROVED** [review]

**Comment by ar-siddiqui** [comment]
@albrazeau all changes look good but with this approach, each fix only fixes one endpoint, so now upsert Geom endpoint still uses the `NewRasModel` function thus taking 2x times than needed.

And this is actually the endpoint that we utilize in `MCAE` on dms.

**Review by ar-siddiqui — APPROVED** [review]

**Pull request merged**